dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -184,6 +184,13 @@ sslCert.option-desc.keyType=\
 \tfrequently the format provided by certificate authorities when\n\
 \tissuing certificates.
 
+sslCert.option-key.ext=\
+\ \ \ \ --extInfo=certificate extension information
+sslCert.option-desc.ext=\
+\tCertificate extension information to add to the certificate.\n\
+\tThe default extension will be a Subject Alternative Name entry using\n\
```

Comment:
Recommend changing:
The default extension will be a Subject Alternative Name entry using
to
The default extension is a Subject Alternative Name entry using

Reply:
Thank you @jantley-ibm!   I've made the suggested change.